dor/providers/package_generator.py

```diff
@@ -166,6 +169,17 @@ def incorporate_file_sets(self, physical_struct_map: StructMap) -> Tuple[list[st
                 missing_file_set_ids.append(file_set_id)
         return incorporated_file_set_ids, missing_file_set_ids
 
+    def search_for_file_sets(self, file_set_ids: list[str]) -> Tuple[list[str], list[str]]:
+        referenced_file_set_ids: list[str] = []
+        missing_file_set_ids: list[str] = []
+        for file_set_id in file_set_ids:
+            search_result = self.op_client.search_for_file_set(file_set_id)
```

Comment:
Assuming that the API is returning _all bins that reference this file_set_id_ then it's possible this call will get back a list of bins: the bin with `file_set_id`, and the bins that have the clone of `file_set_id`.

The results will need to be filtered by

- is my bin in this list?
- which bin actually has `file_set_id`?